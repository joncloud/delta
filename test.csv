2023-05-08T19:49:00.000Z,a
2023-05-08T19:49:15.000Z,b
2023-05-08T19:49:15.863Z,c
