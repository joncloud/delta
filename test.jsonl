{"time":"2023-05-08T19:49:00.000Z","data":"a"}
{"time":"2023-05-08T19:49:15.000Z","data":"b"}
{"time":"2023-05-08T19:49:15.863Z","data":"c"}
