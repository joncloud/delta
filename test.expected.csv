0,2023-05-08T19:49:00.000Z,a
15.000000,2023-05-08T19:49:15.000Z,b
0.863000,2023-05-08T19:49:15.863Z,c
